content/v2.0/security/enable-https.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Enable HTTPS with InfluxDB
+description: >
+  Enable Transport Layer Security (TLS) and use the HTTPS protocol to secure communication between clients and InfluxDB.
+weight: 101
+menu:
+  v2_0:
+    name: Enable HTTPS
```

Comment:
If you change the title to `Enable TLS encryption`, just remove the `name` line. It will default to the `title`.